test/xpu_api/random/interface_tests/engines_methods.pass.cpp

```diff
@@ -248,8 +247,6 @@ test
 public:
     bool run(sycl::queue& queue)
     {
-        using result_type = typename Engine::scalar_type;
```

Comment:
The same for that - it is a simple check of API.

Reply:
Fixed